### PR TITLE
Infra/docs site

### PR DESCRIPTION
**Pull request by Inbal-Tish**
Adding doc site to the repo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,7 @@ npm-debug.log
 yarn.lock
 .vscode
 package-lock.json
-
+docs/**/*.md
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply .gitignore:7]: Why all docsRNC folder should be ignored?

[the remark this answers is not in the document]
Comment (ethanshar) [reply .gitignore:7]: Cause it's not needed when installed in node_modules

File: demo/docsRNC/blog/2019-05-28-first-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply demo/docsRNC/blog/2019-05-28-first-blog-post.md]: Done

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply .gitignore:7]: Right. Done

File: scripts/build-docs.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+const _ = require('lodash');
+const childProcess = require('child_process');
+const fs = require('fs');
+
+const COMPONENTS_DOCS_DIR = './demo/docsRNC/docs';
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply scripts/build-docs.js]: Done

File: .gitignore [hunk 4]
```diff
@@ -4,7 +4,7 @@ npm-debug.log
 yarn.lock
 .vscode
 package-lock.json
-
+docsRNC/
```
Comment (ethanshar) [inline .gitignore:7]: You probably meant to add this to `.npmignore`

**Review by ethanshar — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (Inbal-Tish) [reply .gitignore:7]: correct

**Review by ethanshar — APPROVED** [review]

**Pull request merged**